edu.harvard.i2b2.data/Release_1-7/NewInstall/Metadata/data_build.xml

```diff
@@ -133,10 +133,10 @@ END;
 		</unzip>
 
 		<sql driver="${db.driver}" url="${db.url}" userid="${db.username}" password="${db.password}" classpathref="classpath" onerror="continue">
-			<transaction src="./${db.project}/scripts/schemes_insert_data.sql" />
+			<transaction src="./${db.project}/scripts/schemes_insert_data_${db.type}.sql" />
```

Comment:
again, better to follow convention scripts/[dbtype]/scriptname.sql

Reply:
Fixed